File: messageGo/httpData/agentServiceData.go
package httpData

type AgentServiceResp struct {
	ErrorCode    int32  `json:"errorCode"` //TODO: NEED USE GLOBAL ERROR CODE
	ErrorMessage string `json:"errorMessage"`
	Url          string `json:"url"`
	Online       int32  `json:"online"`
	MaxOnline    int32  `json:"maxOnline"`
	CreatedAt    int64  `json:"createdAt"`
	UpdateAt     int64  `json:"updatedAt"`
}

File: messageDotNet/httpData/AgentServiceData.cs
using System;
using UnityEngine;

/// <summary>
/// http请求网关数据的response
/// </summary>
[Serializable]
public class AgentServiceResp
{
    public int ErrorCode;//TODO: NEED USE GLOBAL ERROR CODE
    public string ErrorMessage;
    public string Url;
    public int Online;
    public int MaxOnline;
    public long CreatedAt;
    public long UpdateAt;

    public string ToJson()
    {
        return JsonUtility.ToJson(this);
    }

}